tabular/src/autogluon/tabular/models/tab_transformer/hyperparameters/parameters.py

```diff
@@ -0,0 +1,69 @@
+from ....constants import BINARY, MULTICLASS, REGRESSION
+
+
+def get_fixed_params():
+    """ Parameters that currently cannot be searched during HPO
+    TODO: HPO NOT CURRENTLY IMPLEMENTED FOR TABTRANSFORMER
+    Will need to figure out what (in future PR) is "fixed" and what is searchable. """
+    fixed_params = {'batch_size': 512,
```

Comment:
short comment to explain each hyperparameter. Also would be good to list possible values of each hyperparameter.

Reply:
Added from TT paper.